browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -132,8 +110,8 @@ class AdblockCnameResolveHostClient : public network::mojom::ResolveHostClient {
     cb_ = base::BindOnce(&ShouldBlockAdWithOptionalCname, task_runner,
                          std::move(next_callback), ctx);
 
-    auto* web_contents = GetWebContents(
-        ctx->render_process_id, ctx->render_frame_id, ctx->frame_tree_node_id);
+    auto* web_contents =
```

Comment:
we probably can drop web_contents at all because it is only used for getting context, which is already stored in `ctx`. Probably a follow-up though

Reply:
As fas as I can see is also used for deciding whether to early return if no WebContents can be obtained (e.g. in case the associated RFH is already shutting down), so I'll leave it for now if it's ok.

Reply:
@mariospr I think that's mainly because of https://github.com/brave/brave-core/pull/8307/commits/465000be70334091efd234cb42d5b3d3213668ab#diff-0986ef9872f15c706dc15f78ee8ef549348a10918b13d126e4483261d7171853R121 which doesn't appear to be used anyway. We don't actually care about exiting early if the web_contents is null

Reply:
Hmm.. ok, I think it's probably safe then to ignore that (webcontents not used anyway other than for getting the context, as Ivan said, so no null dereferences in sight). Fixed in https://github.com/brave/brave-core/commit/79727f64079779b1afb8ef8b3a62902cd2e928e5 (run browser and unit tests locally too, just in case)